src/Npgsql/Replication/ReplicationConnection.cs

```diff
@@ -587,7 +587,7 @@ async Task SendFeedback(bool waitOnSemaphore = false, bool requestReply = false,
                 buf.WriteInt64(Interlocked.Read(ref _lastReceivedLsn));
                 buf.WriteInt64(Interlocked.Read(ref _lastFlushedLsn));
                 buf.WriteInt64(Interlocked.Read(ref _lastAppliedLsn));
-                buf.WriteInt64(TimestampHandler.ToPostgresTimestamp(DateTime.Now));
+                buf.WriteInt64(new NpgsqlTimestamp(default).Microseconds);
```

Comment:
Not sure this is correct

Reply:
Defaults you see here are just todos which should be fixed, the PR isn't ready for the final review, but for an API design review it's fine.